packages/dds/tree/src/feature-libraries/chunked-forest/codec/compressedEncode.ts

```diff
@@ -430,12 +430,13 @@ export class IncrementalChunkShape extends ShapeGeneric<EncodedChunkShape> {
 	 */
```

Comment:
Above, can you link the actual shape being used, which I assume is supposed to be `EncodedInlineArrayShape`?

That said, this function does not seem to use that shape, since using EncodedInlineArrayShape requies the shape to contain the count, and there is no way for the caller of this to know or get the count.

Also it would be good to clarify in those docs (and likley the function name as well) that this `encodeChunk` encodes the referenced contents of the chunk (the root of the seperate incrementally encoded blob) not the references to them (which is what the rest of `IncrementalChunkShape` deals with)